tests/unit/test_image.py

```diff
@@ -8,32 +28,50 @@
     read_image,
     write_image,
 )
-from toolviper.utils.data import download
+from xradio.image._util._casacore.common import _create_new_image as create_new_image
+from xradio.image._util._casacore.common import _open_image_ro as open_image_ro
 from xradio.image._util.common import _image_type as image_type
 from xradio.image._util._casacore.common import _object_name
 
 from xradio.image._util._casacore.common import (
     _open_image_ro as open_image_ro,
     _create_new_image as create_new_image,
 )
-from xradio._utils._casacore.tables import open_table_ro
-
-import dask.array as da
-from glob import glob
-import numbers
-import numpy as np
-import numpy.ma as ma
-import os
-import importlib.resources
-import shutil
-import sys
-import unittest
-import xarray as xr
-import copy
+from toolviper.dask.client import local_client
 
 sky = "SKY"
 
 
+@pytest.fixture(scope="module")
+def dask_client_module():
+    """Set up and tear down a Dask client for the test module.
+
+    This fixture starts a local Dask cluster with specified resources before
+    any tests in the module are run, and ensures the client and cluster are
+    properly closed after all tests complete.
+
+    Returns
+    -------
+    distributed.Client
+        A Dask client connected to a local cluster, shared across all tests
+        in the module.
+    """
+    print("\nSetting up Dask client for the test module...")
+    client = local_client(cores=4, serial_execution=False)
```

Comment:
With this fixture, the read/write tests will run along with a local dask cluster.